### PR TITLE
feat: Added required fields and schemas for ngo events

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Part of #5991 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
- Adds fields in schemas and models for NGO events
- Added migration

#### Changes proposed in this pull request:

- Added migration
- Modified fields

### CODE REVIEW

File: migrations/versions/c517c6be040d_.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'c517c6be040d'
+down_revision = '76aed36b4888'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_ngo', sa.Boolean(), nullable=True))
```
Comment (iamareebjamal) [inline migrations/versions/c517c6be040d_.py]: nullable=False

Comment (mrsaicharan1) [reply migrations/versions/c517c6be040d_.py]: > RC

Updated.

**Comment by iamareebjamal** [comment]
The migration will fail

**Comment by mrsaicharan1** [comment]
> The migration will fail

Is it essential to mention `nullable=False` in SQLAlchemy model as well?

**Comment by iamareebjamal** [comment]
Yes, but not related to migration. The migration will fail because default value is not supplied in the migration

**Pull request merged**